python/minimal_connection.py

```diff
@@ -62,7 +57,7 @@ def main():
 
     for obj in tStream:
         print obj
-        pass
+#        pass
```

Comment:
If you want i can just delete minimal_connection.py and keep it outside of tree... it does nothing useful-- it is purely a scaffolding to develop the live telemetry interface with (and that i end up playing with as development continues)